lib/kb_hisat2/file_util.py

```diff
@@ -96,38 +96,25 @@ def fetch_reads_refs_from_sampleset(ref, ws_url, callback_url):
     """
     obj_type = get_object_type(ref, ws_url)
     refs = list()
-    if "KBaseSets.ReadsSet" in obj_type:
+    if "KBaseSets.ReadsSet" in obj_type or "KBaseRNASeq.RNASeqSampleSet" in obj_type:
         print("Looking up reads references in ReadsSet object")
-        set_client = SetAPI(callback_url)
+        set_client = SetAPI(srv_wiz_url, service_ver='dev')
```

Comment:
Can we advance the changes we need in SetAPI rather than pin to the dev version? I'm concerned that this would not give us much leeway to test further changes to SetAPI without affecting the function of released apps

Reply:
Done